app/src/main/AndroidManifest.xml

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="io.ooni.mk.nuvolari.nuvolari"
-          xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.ooni.mk.nuvolari.android">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:allowBackup="true"
+        android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:ignore="GoogleAppIndexingWarning,UnusedAttribute">
```

Comment:
What's this?

Reply:
This just suppresses some tooling warnings that are not important.  App Indexing is not relevant for this demo app.  I can remove the unused attribute since that was a temporary change while I was iterating.

Reply:
> I can remove the unused attribute [...]

Yes, please!